modules/storage/config.go

```diff
@@ -37,7 +36,6 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 
 	cfg.Trace.WAL = &wal.Config{}
 	f.StringVar(&cfg.Trace.WAL.Filepath, util.PrefixConfig(prefix, "trace.wal.path"), "/var/tempo/wal", "Path at which store WAL blocks.")
-	cfg.Trace.WAL.Version = v2.VersionString
```

Comment:
The ability to configure the wal separately from the backend was useful for iteration and experimentation.  What is the rationale to remove it?

Reply:
Agreed. I have restored `wal.version` which defaults to `block.version` if there's an empty string. It is purposefully undocumented.